cmd/utils.go

```diff
@@ -105,6 +105,12 @@ const (
 	httpsScheme = "https"
 )
 
+// nopCharsetConvert is a dummy charset convert which just copies input to output,
```

Comment:
Typo `nopCharsetConvert` should be `nopCharsetConverter`

Reply:
> Typo `nopCharsetConvert` should be `nopCharsetConverter`

Fixed